tests/data_asset/test_data_asset_util.py

```diff
@@ -89,16 +90,12 @@ def test_recursively_convert_to_json_serializable():
         )
 
     # TypeError when non-serializable numpy object is in dataset.
+    temp_dir = tmp_path / "numpy_non_serializable"
+    temp_dir.mkdir()
     with pytest.raises(TypeError):
-        y = {"p": np.DataSource()}
+        y = {"p": np.DataSource(temp_dir)}
         ge.data_asset.util.recursively_convert_to_json_serializable(y)
-
-    try:
-        x = unicode("abcdefg")
-        x = ge.data_asset.util.recursively_convert_to_json_serializable(x)
-        assert isinstance(x, unicode)
-    except NameError:
-        pass
+    shutil.rmtree(temp_dir)
```

Comment:
@Aylr - we might be able to remove this `rmtree` call since we are using the function scoped `tmp_path`

Reply:
Agreed.

Reply:
Fixed.